bin/run_estimate.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo "Usage: $0 /path/to/ocns"
+run_generic_job.sh bundle exec ruby bin/compile_estimated_IC_costs.rb $1
```

Comment:
This looks like it will be CWD-sensitive. Should it be? Also, should it have any error output if $1 is missing?

Reply:
:shrug: Probably not, and probably. Good catch.